pom.xml

```diff
@@ -257,6 +250,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses />.
         <sonar.host.url>http://127.0.0.1:8080</sonar.host.url>
         <sonar.projectName>lightblue-rest</sonar.projectName>
         <rpm.install.basedir>/usr/share/jbossas/standalone/deployments</rpm.install.basedir>
+        <guava>18.0</guava>
```

Comment:
Because of how the release plugin works it's preferred to put the version in the dependency directly instead of in a property.